content/quantum_coin_game/quantum_coin_game.ipynb

```diff
@@ -0,0 +1,3911 @@
+      "text/plain": [
+       "<Figure size 504x360 with 1 Axes>"
+      ]
+     },
+     "execution_count": 95,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "backend_real = provider.get_backend('ibmq_manila')\n",
+    "job_real = execute(qc, backend_real, shots=8192)\n",
+    "res_real = job_real.result().get_counts()\n",
+    "print(res_real)\n",
+    "plot_histogram(res_real)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### **Quantum Computer Wins all the time.**"
```

Comment:
```suggestion
    "Unlike the perfect simulation, the real quantum computer only wins ~$99\\%$ of the time, the $1\\%$ in which it loses is due to errors. Quantum computers have improved a bit since [Shohini's talk](#conclusion) where the error is closer to $3\\%$."
```